clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheStateServiceImpl.java

```diff
@@ -242,6 +250,139 @@ private ResourcePageSource createPageSource(String poolName, ServerSideConfigura
     return pageSource;
   }
 
+  private void registerStoreStatistics(String storeName) throws InvalidStoreException {
+
+    ServerStoreImpl store = stores.get(storeName);
+    if (store == null) {
+      throw new InvalidStoreException("registerStoreStatistics, Clustered tier '" + storeName + "' does not exist");
```

Comment:
Scratch what I said - just pass the `Store` instance into the method instead of a name that you have to look up. And then you do not even have to ask the question.